docs/content/manual/manual.md

```diff
@@ -0,0 +1,5538 @@
+
+~~~
+.[]|numbers
+[[],{},1,"foo",null,true,false]
+1
+~~~
+
+:::
+
+## Membership functions
+
+### `contains(element)` {#contains}
+
+The filter `contains(b)` will produce true if b is
+completely contained within the input. A string B is
+contained in a string A if B is a substring of A. An array B
+is contained in an array A if all elements in B are
+contained in any element in A. An object B is contained in
+object A if all of the values in B are contained in the
+value in A with the same key. All other types are assumed to
+be contained in each other if they are equal.
+
+::: Examples
+
+~~~
+contains("bar")
+"foobar"
+true
+~~~
+
+~~~
+contains(["baz", "bar"])
+["foobar", "foobaz", "blarp"]
+true
+~~~
+
+~~~
+contains(["bazzzzz", "bar"])
+["foobar", "foobaz", "blarp"]
+false
+~~~
+
+~~~
+contains({foo: 12, bar: [{barp: 12}]})
+{"foo": 12, "bar":[1,2,{"barp":12, "blip":13}]}
+true
+~~~
+
+~~~
+contains({foo: 12, bar: [{barp: 15}]})
+{"foo": 12, "bar":[1,2,{"barp":12, "blip":13}]}
+false
+~~~
+
+:::
+
+### `indices(s)` {#indices}
+
+Outputs an array containing the indices in `.` where `s`
+occurs.  The input may be an array, in which case if `s` is an
+array then the indices output will be those where all elements
+in `.` match those of `s`.
+
+::: Examples
+
+~~~
+indices(", ")
+"a,b, cd, efg, hijk"
+[3,7,12]
+~~~
+
+~~~
+indices(1)
+[0,1,2,1,3,1,4]
+[1,3,5]
+~~~
+
+~~~
+indices([1,2])
+[0,1,2,3,1,4,2,5,1,2,6,7]
+[1,8]
+~~~
+
+:::
+
+### `index(s)`, `rindex(s)` {#index-rindex}
+
+Outputs the index of the first (`index`) or last (`rindex`)
+occurrence of `s` in the input.
+
+::: Examples
+
+~~~
+index(", ")
+"a,b, cd, efg, hijk"
+3
+~~~
+
+~~~
+index(1)
+[0,1,2,1,3,1,4]
+1
+~~~
+
+~~~
+index([1,2])
+[0,1,2,3,1,4,2,5,1,2,6,7]
+1
+~~~
+
+~~~
+rindex(", ")
+"a,b, cd, efg, hijk"
+12
+~~~
+
+~~~
+rindex(1)
+[0,1,2,1,3,1,4]
+5
+~~~
+
+~~~
+rindex([1,2])
+[0,1,2,3,1,4,2,5,1,2,6,7]
+8
+~~~
+
+:::
+
+### `inside`
+
+The filter `inside(b)` will produce true if the input is
+completely contained within b. It is, essentially, an
+inversed version of `contains`.
+
+::: Examples
+
+~~~
+inside("foobar")
+"bar"
+true
+~~~
+
+~~~
+inside(["foobar", "foobaz", "blarp"])
+["baz", "bar"]
+true
+~~~
+
+~~~
+inside(["foobar", "foobaz", "blarp"])
+["bazzzzz", "bar"]
+false
+~~~
+
+~~~
+inside({"foo": 12, "bar":[1,2,{"barp":12, "blip":13}]})
+{"foo": 12, "bar": [{"barp": 12}]}
+true
+~~~
+
+~~~
+inside({"foo": 12, "bar":[1,2,{"barp":12, "blip":13}]})
+{"foo": 12, "bar": [{"barp": 15}]}
+false
+~~~
+
+:::
+
+### `has(key)` {#has}
+
+The builtin function `has` returns whether the input object
+has the given key, or the input array has an element at the
+given index.
+
+`has($key)` has the same effect as checking whether `$key`
+is a member of the array returned by `keys`, although `has`
+will be faster.
+
+::: Examples
+
+~~~
+map(has("foo"))
+[{"foo": 42}, {}]
+[true, false]
+~~~
+
+~~~
+map(has(2))
+[[0,1], ["a","b","c"]]
+[false, true]
+~~~
+
+:::
+
+### `in`
```

Comment:
`in` is arity 1, so maybe `in(x)` should be correct.

Reply:
Done.